spec/migrations/20180828092111_add_parent_physical_chassis_id_to_physical_chassis_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_migration
```

Comment:
we don't need specs for structural migrations, they are tested by rails's specs, we need it only for migration with data manipulation, so you can remove it, thanks!